releasenotes/notes/go-1-13-8-d98857855401db29.yaml

```diff
@@ -0,0 +1,4 @@
+---
+other:
+  - |
+    The Agent, Logs Agent and the system-probe are now compiled with Go ``1.13.8``
```

Comment:
Also APM and process agents no?

(in that case, I guess writing `all the Agent binaries` makes sense)

Reply:
Got a bit too much excited by seeing all those ✅
Let me send another quick PR for that...